code/modules/reagents/chemistry/reagents/cat2_medicine_reagents.dm

```diff
@@ -417,7 +417,7 @@
 		var/mob/living/carbon/carbies = M
 		if (carbies.stat == DEAD)
 			show_message = 0
-		if(method in list(PATCH, TOUCH, VAPOR))
+		if(method & (PATCH|TOUCH|VAPOR))
```

Comment:
i dont think this is valid syntax

Reply:
According to MOMMI it is both valid syntax and faster than building a list just to check a bitflag.

Reply:
#52927 has a few examples.

Reply:
woops i thought i deleted this comment, but yeah i checked its valid.

Reply:
Thanks.